### PR TITLE
Responsify the main page

### PR DESCRIPTION
**Pull request by nienkedekker**
This PR will make the main page responsive -- login and register forms will be added in a separate branch.

### CODE REVIEW

File: public/css/main.css [hunk 1]
```diff
@@ -1,12 +1,11 @@
 @import url(normalize.css);
 @import url(flaticon.css);
 @import url(tables.css);
-@import url(https://fonts.googleapis.com/css?family=Open+Sans|Roboto|Playfair+Display|Bree+Serif|Lato|Source+Sans+Pro|Montserrat|Inconsolata);
-@import url(https://fonts.googleapis.com/css?family=Karla|Raleway|Bitter|Quicksand|Patua+One|Rubik|Cantarell);
+@import url(https://fonts.googleapis.com/css?family=Karla|Montserrat|Inconsolata|Rubik);
```
Comment (EaterOfCode) [inline public/css/main.css]: when are we gonna add those to the repo, me no like 3rd party content

File: public/css/main.scss [hunk 2]
```diff
@@ -81,34 +107,30 @@ body {
...
 
 // third section containing the footer
 .footer-page {
+  @extend .flexpage;
   background-color: #e2f2f0;
-  display: flex;
   flex-direction: column;
-  justify-content: center;
-  align-items: center;
+  height: 20vh;
```
Comment (EaterOfCode) [inline public/css/main.scss]: why did you add this again?  this breaks on low-height screens :o

File: public/css/main.scss [hunk 3]
```diff
@@ -19,57 +35,67 @@
...
   @include long-shadow-text(30, 0, #F1D4AF);
-}
-
-.fa-angle-double-down {
-  color: #424769;
-  margin-top: 50px;
+  @include break-small {
+   font-size: 3.5em;
+   padding: 20px;
+   @include long-shadow-text(20, 0, #F1D4AF);
```
Comment (EaterOfCode) [inline public/css/main.scss]: Don't know how it looks on your phone but it stills look a bit too long for me

File: public/css/main.css (hunk 1 above)
Comment (nienkedekker) [inline public/css/main.css]: it's just google fonts though..

File: public/css/main.scss (hunk 2 above)
Comment (nienkedekker) [inline public/css/main.scss]: Fixed this locally, do I now make a new commit in a new branch?

File: public/css/main.scss (hunk 3 above)
Comment (nienkedekker) [inline public/css/main.scss]: How do I test localhost stuff on my phone :(

File: public/css/main.scss (hunk 2 above)
Comment (EaterOfCode) [inline public/css/main.scss]: you can add this to this branch and make a new PR

File: public/css/main.scss (hunk 3 above)
Comment (EaterOfCode) [inline public/css/main.scss]: if your phone and laptop on the same network and you're not on a VPN you can access the dev-server via `http://[laptop local ip]:8888/` if thats not possible, like at work or so, you can use Google Chrome's device toolbar

File: public/css/main.css (hunk 1 above)
Comment (EaterOfCode) [inline public/css/main.css]: I know, but any 3rd party site can then track the usage of this site. which is something I want to avoid, if you don't feel like moving it to the repo, I'm fine with doing it
